src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.DecisionDagRewriter.cs

```diff
@@ -831,6 +844,39 @@ void lowerFloatDispatch(int firstIndex, int count)
                         }
                     }
                 }
+
+                return;
+
+                bool hasLengthBasedDispatchRequiredMembers(StringPatternInput stringPatternInput)
+                {
+                    var lengthMember = stringPatternInput switch
+                    {
+                        StringPatternInput.String => _localRewriter._compilation.GetSpecialTypeMember(SpecialMember.System_String__Length),
```

Comment:
>_localRewriter._compilation [](http://example.com/codeflow?start=53&length=27)

Consider extracting a local, or pass in the compilation as a parameter and make the local function `static`.